random.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CodersCamp2020 | CookBook</title>
+    <link rel="stylesheet" href="styles/App.css" />
+
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <div id="swquiz-app">
+    </div>
+    <div id="contentRandom">
+
+
+        <button id="random">Random</button>
```

Comment:
Can we change this id to for example randomButton?

Reply:
Ok